### PR TITLE
Revert #5772

### PR DESCRIPTION
**Pull request by Gargron**
https://github.com/tootsuite/mastodon/pull/5649#issuecomment-355847223

### CODE REVIEW

File: spec/models/follow_request_spec.rb [hunk 1]
```diff
@@ -1,38 +1,25 @@
...
+      expect(follow_request).to model_have_error_on_field(:account)
     end
 
-    it 'correctly passes show_reblogs when false' do
-      follow_request = Fabricate.create(:follow_request, show_reblogs: false)
-      follow_request.authorize!
-      target = follow_request.target_account
-      expect(follow_request.account.muting_reblogs?(target)).to be true
-    end
-  end
```
Comment (nightpool) [inline spec/models/follow_request_spec.rb]: these removals seem unrelated?

Comment (nightpool) [inline spec/models/follow_request_spec.rb]: these removals seem unrelated?

**Review by nightpool — CHANGES_REQUESTED** [review]
LGTM with spec changes cleaned up (some show_reblog changes got caught in the cross-fire)

**Review by nightpool — APPROVED** [review]

**Comment by akihikodaki** [comment]
This should be merged as soon as possible if #6211 is making Mastodon incompatible with Kroeg, and later we can investigate the behavior should be fixed.
Otherwise, #6224 should be merged instead of this.
